l10n_es_facturae/models/account_journal.py

```diff
@@ -24,7 +24,12 @@ def fields_view_get(
             if node:
                 return res
             for node in doc.xpath("//field[@name='type']"):
-                elem = etree.Element("field", {"name": "thirdparty_invoice"})
+                attrs = {
+                    "invisible": [("type", "not in", ("sale", "purchase"))],
+                }
+                elem = etree.Element(
+                    "field", {"name": "thirdparty_invoice", "attrs": str(attrs)}
+                )
                 node.addnext(elem)
```

Comment:
no need to call
```python
transfer_node_to_modifiers(elem, modifiers)
transfer_modifiers_to_node(modifiers, elem)
```
 here?

Reply:
En este caso no es necesario porque luego llamamos a la función `postprocess_and_fields()` ; únicamente es necesario usar las funciones
```
transfer_node_to_modifiers(elem, modifiers)
transfer_modifiers_to_node(modifiers, elem)
```

para la vista de asientos para evitar que se hagan cambios en la vista que no queremos.